tensorflow_datasets/image_classification/rock_paper_scissors.py

```diff
@@ -40,7 +40,7 @@
 _IMAGE_SIZE = 300
 _IMAGE_SHAPE = (_IMAGE_SIZE, _IMAGE_SIZE, 3)
 
-_NAME_RE = re.compile(r"^(rps|rps-test-set)/(rock|paper|scissors)/[\w-]*\.png$")
+_NAME_RE = re.compile(r"^(rps|rps-test-set)(/|\\)(rock|paper|scissors)(/|\\)[\w-]*\.png$")
```

Comment:
There shouldn't be brace as this indicates a capture group. Will fix this internally.